website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -16,13 +16,45 @@ A secure Consul cluster should include the following:
 
 -> **NOTE:** This page assumes that you have already configured your Consul server with the above features.
 
+## Auth Method
+
+Consul on ECS uses the [AWS IAM Auth Method](/docs/security/acl/auth-methods/aws-iam) to enable ECS
+tasks to automatically obtain Consul ACL tokens during startup. See the [Architecture](/docs/ecs/architecture)
+page for an overview of task startup, task shutdown, ACL tokens, and the AWS IAM auth method for Consul on ECS.
+
+### Auth Method Configuration
+
+When using the Terraform modules for Consul on ECS, the auth method is automatically configured
+by the ACL controller, and ECS tasks will be configured by the `mesh-task` module to be able
+to log in to the auth method.
+
+The `mesh-task` module configures the task IAM role as follows:
```

Comment:
Do they need to ensure each service task family has a different task iam role?

Reply:
Should we talk about how the mesh task module accepts a `task_role`  variable. If not passed, then we will automatically create valid roles.

If passed, then we will add tags to those roles.